src/pudl/analysis/record_linkage/eia_ferc1_record_linkage.py

```diff
@@ -587,6 +604,18 @@ def run_model(
         f"    Precision: {precision:.02}\n"
         f"    Recall:    {recall:.02}\n"
     )
+    # Log model metrics
+    experiment_tracker.execute_logging(
```

Comment:
I think ultimately these won't be the metrics that we track but this model will likely get replaced by the splink model so let's make those changes in the splink PR

Reply:
It doesn't has to be in this PR, but it would be probably be good to update the metrics while this model is in use, so we can compare them to splink once it's in place.

Reply:
Yep, I'll do this in the splink PR so we can more easily compare results.